htdocs/api/v0.0.2/Candidates.php

```diff
@@ -113,18 +113,21 @@ public function handlePOST()
             // This version od the API does not handle candidate creation 
             // when users are at multiple sites
             $user = \User::singleton();
-            $centerIDs = $user->getData('CenterIDs');
+            $centerIDs = $user->getCenterIDs();
             $num_sites = count($centerIDs);
 
-            if ($num_sites >1) {
+            if ($num_sites == 0) {
+                $this->header("HTTP/1.1 401 Unauthorized");
+                $this->error("You are not affiliated with any site");
+                $this->safeExit(0);
+            } else if ($num_sites > 1) {
                 $this->header("HTTP/1.1 501 Not Implemented");
                 $this->error("This API version does not support timepoint creation " .
                               "by uers with multiple site affilifations. This will be ".
```

Comment:
@AnyhowStep 

same 2 comments as above (spelling and 'timepoint creation' should be 'candidate creation')